e2e-test/cromwell_azure_e2e_test.py

```diff
@@ -66,13 +66,15 @@ def get_workflow_metadata(app_url, workflow_id, bearer_token):
     return perform_get(workflow_endpoint, bearer_token)
 
 # workflow_ids is a deque of workflow ids
-def get_completed_workflow(app_url, workflow_ids, bearer_token, max_retries=4, sleep_timer=60 * 2):
+def get_completed_workflow(app_url, workflow_ids, bearer_token, max_retries=6, sleep_timer=60 * 2):
     success_statuses = ['Succeeded']
     throw_exception_statuses = ['Aborted', 'Failed']
+
+    current_retries = max_retries
     
     current_running_workflow_count = 0
     while workflow_ids:
-        if max_retries == 0:
+        if current_retries == 0:
```

Comment:
Makes sense, I would personally expect a value called "current_retries" to increase as we retry more... Maybe "remaining_retries" would be slightly nicer?